ivadomed/scripts/training_curve.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python
+
+import os
+import argparse
+from collections import defaultdict
+from tensorflow.python.summary.summary_iterator import summary_iterator
+import pandas as pd
+import matplotlib.pyplot as plt
+
+DEBUGGING = False
+
+if DEBUGGING:
+    import matplotlib
+    matplotlib.use('TkAgg')
+
+
+def get_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-i", "--input", required=True, type=str,
+                        help="Input log directory. If using -m True, this parameter indicates the suffix path of all "
+                             "log directories of interest.")
+    parser.add_argument("-m", "--multiple", required=False, default=False, type=bool,
+                        help="If True, then multiple log directories are considered: all available folders with -i as "
+                             "prefix. The plot represents the mean value (hard line) surrounded by the standard "
+                             "deviation envelope.")
```

Comment:
I thought of this and then was hesitant because I was scared that we would not see much by overlapping multiple graphs + SD. That's why I opted to generate graphs separetly. The good thing with our application is that the scale is the same, so easy comparison.
An alternative would be to have something like `-i folder/prefix1,folder/prefix2` --> and plot the two folders' results in two subplots one next to the other one (instead of all overlapped).
What are your thoughts @andreanne-lemay ?